### DISCUSSION TITLE
MET/METPlus build in Derecho/ Casper

### DISCUSSION THREAD

**Discussion by zhumingying** [dtcenter/METplus#2483] [Existing Builds] [answered]
Could anyone let me know where the MET/METPlus build in Derecho/
Casper? /glade/p/ral/jntp/MET/METplus/modulefiles is not exist

**Comment by georgemccabe** [comment]
Hi @zhumingying,

The METplus modules files for derecho can be found in /glade/p/ral/jntp/MET/METplus/**derecho**/modulefiles

```
module use /glade/p/ral/jntp/MET/METplus/derecho/modulefiles
module load metplus/5.1.0
```


The METplus modules files for casper can be found in /glade/p/ral/jntp/MET/METplus/**casper**/modulefiles

It looks like the latest version available for casper is 5.0.1, so I would recommend using derecho if you can.

```
module use /glade/p/ral/jntp/MET/METplus/casper/modulefiles
module load metplus/5.0.1
```

**Comment by jprestop** [comment]
HI @zhumingying.  The locations that @georgemccabe pointed to are locations that are going away and are currently read only.  I had been waiting for the MET-11.1.1 bugfix release to come out before re-installing on Derecho and Casper, but please let us know if you need MET-11.1.0 and the above locations are not working for you and we'll get everything installed sooner rather than later.

**Comment by jprestop** [comment, marked as answer]
Hi @zhumingying.  I have re-installed MET and METplus on both Casper and Derecho.  You can find the appropriate modules to load on our METplus [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-5-1-existing-builds) page under "NCAR machines" for the 5.1 coordinated release.  I hope to have METcalcpy, METdataio, and METplotpy installed there as well by early next week.  Please let us know if you have any questions or problems.
